src/tests/dictionaryConcepts/container/SelectAnswers.test.js

```diff
@@ -138,4 +138,13 @@ describe('<SelectAnswers />', () => {
     expect(wrapper.state().inputValue).toEqual('');
     expect(props.removeCurrentAnswer).toHaveBeenCalled();
   });
+
+  it('should handle change when a user clicks in the search input', () => {
+    wrapper.setState({ hasReset: true });
+    const spy = jest.spyOn(wrapper.instance(), 'handleInputChange');
+    wrapper.find('#searchInputCiel').simulate('change');
+    expect(spy).toHaveBeenCalled();
+    expect(wrapper.instance().state.hasReset).toEqual(false);
```

Comment:
Could we add an assertion before line `145` to make sure this is `true` before calling `simulate`?

Reply:
Done 👍